src/main/fc/tasks.c

```diff
@@ -409,6 +417,10 @@ void tasksInit(void)
 #ifdef USE_RCDEVICE
     setTaskEnabled(TASK_RCDEVICE, rcdeviceIsEnabled());
 #endif
+
+#ifdef USE_CRSF_V3
+    setTaskEnabled(TASK_SPEED_NEGOTIATION, true);
```

Comment:
Enabling this always, even if CRSF V3 isn't configured as the RC protocol doesn't make sense - this should actually check if this task is needed, see `rcdeviceIsEnabled()` above.